packages/icons/README.md

```diff
@@ -1,27 +1,26 @@
-# Vaadin Icons
+# @vaadin/icons
 
-[![npm version](https://badgen.net/npm/v/@vaadin/icons)](https://www.npmjs.com/package/@vaadin/icons)
-[![Published on Vaadin Directory](https://img.shields.io/badge/Vaadin%20Directory-published-00b4f0.svg)](https://vaadin.com/directory/component/vaadinvaadin-icons)
-[![Discord](https://img.shields.io/discord/732335336448852018?label=discord)](https://discord.gg/PHmkCKC)
+A set of 600+ icons designed for web applications.
 
-[<img src="https://raw.github.com/vaadin/vaadin-icons/master/screenshot.png" width="611" alt="Screenshot of some icons in the Vaadin Icons collection" />](https://vaadin.com/icons)
+[Documentation + Live Demo ↗](https://vaadin.com/docs/latest/ds/foundation/icons/vaadin)
 
-[Vaadin Icons](https://vaadin.com/icons) is a set of 600+ icons designed for web applications. Free to use, anywhere!
+[![npm version](https://badgen.net/npm/v/@vaadin/icons)](https://www.npmjs.com/package/@vaadin/icons)
+[![Discord](https://img.shields.io/discord/732335336448852018?label=discord)](https://discord.gg/PHmkCKC)
 
-Visit **https://vaadin.com/icons** for more information and instructions how to get started using them.
+[<img src="https://raw.githubusercontent.com/vaadin/web-components/master/packages/icons/screenshot.png" width="728" alt="Screenshot of vaadin-icons">](https://vaadin.com/docs/latest/ds/foundation/icons/vaadin)
 
 ## Installation
 
-Install `vaadin-icons`:
+Install the iconset:
 
 ```sh
-npm i @vaadin/icons --save
+npm i @vaadin/icons
 ```
 
 Once installed, import it in your application:
 
 ```js
-import '@vaadin/icons/vaadin-icons.js';
+import '@vaadin/icons';
```

Comment:
Could hint that you should use `vaadin-icon` to actually display an icon from this set.

Reply:
Thanks, good point. Added.